src/test/integration/handlers/user_account/GetUserAccountTest.kt

```diff
@@ -0,0 +1,57 @@
+package test.integration.handlers.user_account
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+import io.kotlintest.Description
+import com.amazonaws.services.lambda.runtime.Context
+import com.beust.klaxon.Klaxon
+import io.kotlintest.TestResult
+import kotlinserverless.framework.models.*
+import io.mockk.junit5.MockKExtension
+import org.junit.jupiter.api.extension.ExtendWith
+import io.mockk.mockk
+import main.daos.UserAccount
+import test.TestHelper
+
+@ExtendWith(MockKExtension::class)
+class GetUserAccountTest : WordSpec() {
+    private lateinit var handler: Handler
+    private lateinit var contxt: Context
+    private lateinit var user: UserAccount
+    private lateinit var userAccounts: List<UserAccount>
+    private lateinit var map: Map<String, Any>
+
+    override fun beforeTest(description: Description): Unit {
+        Handler.connectAndBuildTables()
+        user = mockk()
+        handler = Handler(user)
+        contxt = mockk()
+        userAccounts = TestHelper.generateUserAccounts()
+        map = mutableMapOf(
+                Pair("path", "/user_account/"),
+                Pair("httpMethod", "GET"),
+                Pair("pathParameters", mutableMapOf(
+                    Pair("id", userAccounts[0].id.value)
+                ))
+        )
+    }
+
+    override fun afterTest(description: Description, result: TestResult) {
+        Handler.disconnectAndDropTables()
+    }
+
+    init {
+        "correct path" should {
+            "should return a valid user account" {
+                val response = handler.handleRequest(map, contxt)
+                response.statusCode shouldBe 200
+                val newUserAccountMap = Klaxon().parse<Map<String?, Any>>(response.body!!)
+
+                newUserAccountMap!!.containsKey("apiCreds") shouldBe true
+                newUserAccountMap!!.containsKey("session") shouldBe true
+                newUserAccountMap!!.containsKey("cryptoKeyPair") shouldBe true
+                newUserAccountMap!!.containsKey("userMetadata") shouldBe true
```

Comment:
may want to test that this data matches what was generated (maybe just check the email, firstname, lastname) -- you can look in the testhelper to see how those are generated and what should be expected

Reply:
i think this is really important to figure out -- lets do that before merging as it will help us on all future tests